app/Http/Controllers/CustomTourController.php

```diff
@@ -54,6 +54,36 @@ public function show(Request $request, $id)
         ]);
     }
 
+    public function pdfLayout(Request $request, $id)
+    {
+        $customTourItem = CustomTour::find($id);
+
+        if (!$customTourItem) {
+            return abort(404);
+        }
```

Comment:
This can be shortened to `$customTourItem = CustomTour::findOrFail($id);`

See https://laravel.com/docs/10.x/eloquent#not-found-exceptions